grid2op/Rules/RulesChecker.py

```diff
@@ -50,6 +50,17 @@ def __init__(self, legalActClass=AlwaysLegal):
                 warnings.warn("You passed the legal action as an instance that cannot be deepcopied. It will be "
                               "used 'as is', we do not garantee anything if you modify the original object.")
                 self.legal_action = legalActClass
+                
+    def initialize(self, env):
+        """
+        This function is used to inform the class instance about the environment specification. 
+        It can be the place to assert the defined rules are suited for the environement.
+        Parameters
```

Comment:
maybe say the env is not fully initialized when this is called

Reply:
ok !